src/sbuf/MemBlob.cc

```diff
@@ -148,7 +153,7 @@ MemBlob::consume(const size_type rawN)
 const MemBlobStats&
 MemBlob::GetStats()
 {
-    return Stats;
+    return GetStatsInstance();
 }
```

Comment:
Please either move GetStats() definition up or (forward-declare and) move GetStatsInstance() definition down so that the two "give me the stats" functions are defined _next to each other_. I recommend the former because moving GetStats() definition results in more compact code (at the expense of a few additional line modifications).

Reply:
ok